packages/components/.storybook/preview.tsx

```diff
@@ -1,62 +1,14 @@
-import * as React from 'react'
-
-import {
-  LoadingState,
-  ThemeProvider,
-  breakpoints,
-  createCustomShadows,
-  createPalette,
-  createShadows,
-  primaryFont,
-  secondaryFont,
-  typography,
-} from '@baseapp-frontend/design-system'
-import { RelayTestProvider } from '@baseapp-frontend/graphql'
-import { NotificationProvider } from '@baseapp-frontend/utils'
-
-import type { Preview, StoryContext, StoryFn } from '@storybook/react'
+import type { Preview } from '@storybook/react'
 import { initialize, mswLoader } from 'msw-storybook-addon'
 
 import '../styles/tailwind/globals.css'
+import { handlers } from './__mocks__/handlers'
```

Comment:
we could add this css file:

```
// packages/components/.storybook/overrides.css
.simplebar-wrapper {
    height: 64px;
}
```
and import it on the preview file to fix the nav horizontal bug
```
import './overrides.css'
```